src/lib/y2network/sysconfig/connection_config_readers/ethernet.rb

```diff
@@ -17,6 +17,7 @@
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
 
+require "y2network/sysconfig/connection_config_readers/base"
 require "y2network/connection_config/ethernet"
 require "y2network/boot_protocol"
```

Comment:
Added to base reader

Reply:
It may be that I am confused, but I see this line in my code. And same below.

Reply:
It is in the code, I suggest to remove boot_protocol and startmode requirement and add just to the base class.

Reply:
Already done.